test/vcf/4.3/passed/passed_body_info.vcf

```diff
@@ -32,7 +32,7 @@
 6	300	rs180734498	C	T	100	PASS	DB=1	GT:DS:GL	0|0:0.050:-0.13,-0.58,-3.62	0|1:1.000:-2.45,-0.00,-5.00
 7	100	rs180734498	C	T	100	PASS	DP=0	GT:DS:GL	0|0:0.050:-0.13,-0.58,-3.62	0|1:1.000:-2.45,-0.00,-5.00
 7	200	rs180734498	C	T	100	PASS	DP=2184	GT:DS:GL	0|0:0.050:-0.13,-0.58,-3.62	0|1:1.000:-2.45,-0.00,-5.00
-8	100	rs180734498	C	T	100	PASS	END=0	GT:DS:GL	0|0:0.050:-0.13,-0.58,-3.62	0|1:1.000:-2.45,-0.00,-5.00
+8	100	rs180734498	C	T	100	PASS	END=100	GT:DS:GL	0|0:0.050:-0.13,-0.58,-3.62	0|1:1.000:-2.45,-0.00,-5.00
```

Comment:
This would also apply to [test/vcf/4.2/passed/passed_body_info.vcf](https://github.com/samtools/hts-specs/blob/master/test/vcf/4.2/passed/passed_body_info.vcf) and [test/vcf/4.1/passed/passed_body_info.vcf](https://github.com/samtools/hts-specs/blob/master/test/vcf/4.1/passed/passed_body_info.vcf)